lsstseries/analysis/stetsonj.py

```diff
@@ -0,0 +1,139 @@
+import numpy as np
+
+def calc_stetson_J(flux, err, band, band_to_calc=None):
+    """Compute the stetsonJ statistic on data from one or several bands
+        Parameters
+        ----------
+        flux : `numpy.ndarray` (N,)
+            Array of flux/magnitude measurements
+        err : `numpy.ndarray` (N,)
+            Array of associated flux/magnitude errors
+        band : `numpy.ndarray` (N,)
+            Array of associated band labels
+        band_to_calc : `str` or `list` of `str`
+            Bands to calculate stetson J on. Single band descriptor, or list 
+            of such descriptors.
+        Returns
+        -------
+        stetsonJ : `dict`
+            StetsonJ statistic for each of input bands.
+        Notes
+        ----------
+        In case that no value for `band_to_calc` is passed, the function is 
+        executed on all available bands in `band`.
+        """
+
+    unq_band = np.unique(band)
+
+    if band_to_calc is None:
+        band_to_calc = unq_band
+    if isinstance(band_to_calc,str):
+        band_to_calc = [band_to_calc]
+
+    assert hasattr(band_to_calc, '__iter__') is True
+
+    StetsonJ = {}
```

Comment:
As per previous discussion, I suggest changing dictionary name to stetsonJ

Reply:
Done